smoothie-lifecycle/build.gradle

```diff
@@ -30,11 +30,10 @@ android {
 dependencies {
   compileOnly project(':toothpick-runtime')
   api project(':smoothie')
-  implementation deps.inject
```

Comment:
we didn't need it?

Reply:
This module does not contain any JSR annotations